example/config/spire-server.yaml

```diff
@@ -187,10 +203,15 @@ spec:
             - name: spire-config
               mountPath: /run/spire/config
               readOnly: true
+            - name: spire-data
+              mountPath: /run/spire/data
       volumes:
         - name: spire-config
           configMap:
             name: spire-server
+        - name: spire-data
+          persistentVolumeClaim:
+            claimName: spire-server-pvc
```

Comment:
I understand the utility of the PVC, so the SQLite database isn't lost on restarts.

However, the inclusion of a SQLite on a PVC in a Deployment seems odd.  If a person reads this example and then opts to increase the instances, the product will have two pods vying for connections to the same SQLite database.

@abe-hpe If you combine this with a comment near the "instances: 1" line of the Deployment indicating that the data store must be reconfigured for instance counts greater than one, that will fully address my concerns.